modules/reindex/src/main/java/org/elasticsearch/index/reindex/RestReindexAction.java

```diff
@@ -150,11 +150,13 @@ static RemoteInfo buildRemoteInfo(Map<String, Object> source) throws IOException
         String host = hostMatcher.group("host");
         int port = Integer.parseInt(hostMatcher.group("port"));
         Map<String, String> headers = extractStringStringMap(remote, "headers");
+        String timeoutString = extractString(remote, "timeout");
+        TimeValue timeout = timeoutString == null ? RemoteInfo.DEAULT_SOCKET_TIMEOUT : parseTimeValue(timeoutString, "timeout");
```

Comment:
I've renamed it. Do you think I should make `connect_timeout` configurable while I'm here?